package/src/components/AccountProfileInfo/v1/AccountProfileInfo.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { withComponents } from "@reactioncommerce/components-context";
+import { addTypographyStyles, applyTheme } from "../../../utils";
+
+const AccountProfileInfoContainer = styled.div`
+  display: flex;
+  position: relative;
+`;
+
+const AccountProfileInfoTextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+`;
+
+const ViewerEmailText = styled.span`
+  ${addTypographyStyles("ViewerInfoInitials", "labelText")}
+  color: ${applyTheme("accountProfileInfoEmailFontColor")};
+  font-size: ${applyTheme("accountProfileInfoEmailFontSize")};
```

Comment:
`color` and `font-size` are added by `addTypographyStyles` so you should remove those lines.

Also change `ViewerInfoInitials` to `AccountProfileInfoEmail` so that it can be styled separately from ViewerInfo component

Reply:
Do I need to add AccountProfileInfoEmail to anything (the defaultComponentTheme?) or just put the name here, which allows it to be styled later on?